html/captive-portal/templates/wireless-profile.xml

```diff
@@ -6,55 +6,145 @@
         <key>PayloadContent</key>
```

Comment:
As previously discussed. The EAP-TLS provisioning template and the mobile config should not share the same template